recipes/ms_entropy/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "ms_entropy" %}
+{% set version = "1.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ms_entropy-{{ version }}.tar.gz
+  sha256: f6207b6c4013d61d9843938a4cc39be6dfb67c9687288e85ee69511153131c21
+
+build:
+  skip: true  # [py<37]
+  skip: true  # [py<39 and osx]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+  host:
+    - python
+    - setuptools >=61.0
+    - cython >=0.26.1
+    - numpy >=1.9.3
```

Leave this unpinned to get the global pin
```suggestion
    - numpy
```